### ISSUE TITLE
Transport AI refuses direct orders

### ISSUE THREAD

**Issue by Licho1** [ZeroK-RTS/Zero-K#2100]
1) Build dant
2) Build vindicator
3) Give dant move + embark order

Expected: should result in it being picked up and transported 
Actual: nothing happens at all

Screaming, repeating the cycle, restarting luaui and bashing keyboard does not help.

**Comment by GoogleFrog** [comment]
Vindicator has a tactical AI toggle for transport AI (as does Valk). It starts as default off for Vindicator since you often make them for particular tasks, not automated transport spam, but it should be discussed.

**Comment by Licho1** [comment]
I totally didnt expect that, the button on dante said "embark" i clicked it and nothing.. so it was walking and i sent manual vindi to grab it.. because dante didnt stop it just followed the dante across the map..

It felt like game logic failure to me and not an useful feature..

Transport AI can only be used for embark now, unless you guard factory so i think its reasonable to have it on by default on vindi.

**Comment by sprunk** [comment]
>Transport AI can only be used for embark now, unless you guard factory

No, it's also used for constructor ferry - you can set cons to automatically call a transport when moving between build sites.
